storage/fsStorage.go

```diff
@@ -1,20 +1,82 @@
 package storage
 
-import "fmt"
+import (
+	"encoding/json"
+	"errors"
+	"io/ioutil"
+	"os"
+)
 
-type FsStorage struct {
+type fsStorage struct {
 	path string
 }
+type dataBox struct {
+	Value string `json:"value"`
+}
+
+func (st fsStorage) Read(key string) (string, error) {
+	file, err := ioutil.ReadFile(st.path)
+	if err != nil {
+		return "", err
+	}
+	data := make(map[string]dataBox)
+	json.Unmarshal(file, &data)
+	if err != nil {
+		return "", err
+	}
+
+	val, ok := data[key]
+	if ok {
+		return val.Value, nil
+	}
+	return "", errors.New("item was not found")
+}
+
+func (st fsStorage) Write(key string, value string) error {
+	err := checkFile(st.path)
+	if err != nil {
+		return err
+	}
 
-func (st FsStorage) Read(key string) (string, error) {
-	return "got it", nil
+	file, err := ioutil.ReadFile(st.path)
+	if err != nil {
+		return err
+	}
+	data := make(map[string]dataBox)
+	json.Unmarshal(file, &data)
+	if err != nil {
+		return err
+	}
+
+	data[key] = dataBox{Value: value}
+	dataBytes, err := json.Marshal(data)
+	if err != nil {
+		return err
+	}
+
+	err = ioutil.WriteFile(st.path, dataBytes, 0644)
+	if err != nil {
+		return err
+	}
+	return nil
 }
-func (st FsStorage) Write(key string, value string) {
-	fmt.Print(value)
+
+func NewFsStorage() fsStorage {
```

Comment:
It is good to be more explicit here and pass a destination path into the constructor function instead of hardcoding.